airflow/migrations/versions/952da73b5eff_add_dag_code_table.py

```diff
@@ -59,7 +59,7 @@ class SerializedDagModel(Base):
                     sa.Column('last_updated', sa.TIMESTAMP(timezone=True), nullable=False))
 
     conn = op.get_bind()
-    if conn.dialect.name not in ('sqlite'):
+    if conn.dialect.name not in 'sqlite':
```

Comment:
It seems that original code was actually wrong:
```
'sql' in 'sqlite' => True
'sql' in ('sqlite',) => False
```
So, original code should actually be: `.. not in ('sqlite',)`: